patterns/text-faqs.php

```diff
@@ -23,50 +23,61 @@
+			<!-- wp:paragraph {"style":{"typography":{"fontStyle":"normal","fontWeight":"300"}}} -->
+			<p style="font-style:normal;font-weight:300"><?php echo esc_html_x( 'This exquisite compilation showcases a diverse array of photographs that capture the essence of different eras and cultures, reflecting the unique styles and perspectives of each artist. Fleckenstein\'s evocative imagery, Strand\'s groundbreaking modernist approach, and Kōno\'s meticulous documentation of Japanese life come together in a harmonious blend that celebrates the art of photography.', 'Answer in the FAQs pattern.', 'twentytwentyfive' ); ?></p>
+			<!-- /wp:paragraph -->
+
+			<!-- wp:separator -->
+			<hr class="wp-block-separator has-alpha-channel-opacity"/>
+			<!-- /wp:separator -->
+
+			<!-- wp:heading {"level":3} -->
+			<h3 class="wp-block-heading"><?php echo esc_html_x( 'When will The Stories Book be released?', 'Question in the FAQs pattern.', 'twentytwentyfive' ); ?></h3>
+			<!-- /wp:heading -->
+
+			<!-- wp:paragraph {"style":{"typography":{"fontStyle":"normal","fontWeight":"300"}}} -->
+			<p style="font-style:normal;font-weight:300"><?php echo esc_html_x( 'This exquisite compilation showcases a diverse array of photographs that capture the essence of different eras and cultures, reflecting the unique styles and perspectives of each artist. Fleckenstein\'s evocative imagery, Strand\'s groundbreaking modernist approach, and Kōno\'s meticulous documentation of Japanese life come together in a harmonious blend that celebrates the art of photography.', 'Answer in the FAQs pattern.', 'twentytwentyfive' ); ?></p>
+			<!-- /wp:paragraph -->
+
+			<!-- wp:separator -->
+			<hr class="wp-block-separator has-alpha-channel-opacity"/>
+			<!-- /wp:separator -->
+
+			<!-- wp:heading {"level":3} -->
+			<h3 class="wp-block-heading"><?php echo esc_html_x( 'Are signed copies available?', 'Question in the FAQs pattern.', 'twentytwentyfive' ); ?></h3>
+			<!-- /wp:heading -->
+
+			<!-- wp:paragraph {"style":{"typography":{"fontStyle":"normal","fontWeight":"300"}}} -->
+			<p style="font-style:normal;font-weight:300"><?php echo esc_html_x( 'This exquisite compilation showcases a diverse array of photographs that capture the essence of different eras and cultures, reflecting the unique styles and perspectives of each artist. Fleckenstein\'s evocative imagery, Strand\'s groundbreaking modernist approach, and Kōno\'s meticulous documentation of Japanese life come together in a harmonious blend that celebrates the art of photography.', 'Answer in the FAQs pattern.', 'twentytwentyfive' ); ?></p>
+			<!-- /wp:paragraph -->
+
+			<!-- wp:separator -->
+			<hr class="wp-block-separator has-alpha-channel-opacity"/>
+			<!-- /wp:separator -->
+
+			<!-- wp:heading {"level":3} -->
+			<h3 class="wp-block-heading"><?php echo esc_html_x( 'Will The Stories Book be available in other languages other that English?', 'Question in the FAQs pattern', 'twentytwentyfive' ); ?></h3>
```

Comment:
```suggestion
			<h3 class="wp-block-heading"><?php echo esc_html_x( 'Will The Stories Book be available in languages other than English?', 'Question in the FAQs pattern', 'twentytwentyfive' ); ?></h3>
```

The typing mistake is not related to the Details block, but I just noticed that this question should have
- "than" instead of "that" and
- "other" only once,